### PR TITLE
Refactor the activation actions in the UsersController

### PR DESCRIPTION
**Pull request by brambokdam**
The activation actions where stuffed in the UsersController. It looks better if it has it's own
ActivationController.

- Moved the activate and confirm methods from the UsersController to the ActivationController

### CODE REVIEW

File: app/controllers/activation_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class ActivationController < ApplicationController
```
Comment (jvanbaarsen) [inline app/controllers/activation_controller.rb]: What do you think about the name `UserActicationController`?

Comment (brambokdam) [inline app/controllers/activation_controller.rb]: You're right `UserActivationController ` is better, i have changed it. :+1:

**Comment by jvanbaarsen** [comment]
lgtm!

**Comment by jvanbaarsen** [comment]
Thanks :-)

**Pull request merged**